### PR TITLE
Show comment characters length

### PR DESCRIPTION
**Pull request by HanaAyman**
This pull request is to show the maximum length of the comments.
image

### CODE REVIEW

**Review by penk85 — DISMISSED** [review]

File: examples/website/src/shared/molecules/commentTemplate.tsx [hunk 1]
```diff
@@ -202,6 +202,7 @@ export interface LoginToComment {
 
 export function ComposeComment(props: ComposeCommentProps) {
   const css = useStyle()
+  const maxCommentLength = 1000
```
Comment (bishoymelek) [inline examples/website/src/shared/molecules/commentTemplate.tsx:205]: I suggest this to be from an environment variable. what do you think? @HanaAyman

File: examples/website/src/shared/molecules/commentTemplate.tsx [hunk 2]
```diff
@@ -244,7 +260,10 @@ export function ComposeComment(props: ComposeCommentProps) {
             }}
           />
         </div>
-
+        <div style={{float: 'right', fontSize: '0.9em'}}>
+            {' '}
```
Comment (bishoymelek) [inline examples/website/src/shared/molecules/commentTemplate.tsx:264]: Can you clean this one please? for readability

File: examples/website/src/shared/molecules/commentTemplate.tsx [hunk 3]
```diff
@@ -218,6 +219,21 @@ export function ComposeComment(props: ComposeCommentProps) {
     }
   })
 
+  function countRichtextChars(blocksCharLength: number, nodes: any) {
```
Comment (bishoymelek) [inline examples/website/src/shared/molecules/commentTemplate.tsx:222]: I am not sure but i remember we already have a similar function?

**Review by bishoymelek — CHANGES_REQUESTED** [review]

**Review by bishoymelek — APPROVED** [review]
:tada:

**Review by penk85 — APPROVED** [review]

**Pull request merged**